web/src/message_edit.ts

```diff
@@ -594,7 +596,8 @@ function edit_message($row: JQuery, raw_content: string): void {
                 // the half-finished edit around so that they can copy-paste it, but we don't want
                 // people to think "Save" will save the half-finished edit.
                 $message_edit_save.prop("disabled", true);
-                $message_edit_save_container.addClass("tippy-zulip-tooltip");
+                $message_edit_save_container.addClass("times-up");
```

Comment:
This class is added to differentiate when the save button is disabled due to either an editing time expiration or exceeding the maximum length.

Reply:
This class name is too generic. You want something like `message-edit-time-limit-expired` that's really clear and explicit and globally unique across the codebase.